src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_InbreedingCoeffUnitTest.java

```diff
@@ -0,0 +1,218 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
```

Comment:
It's important to have tests be in the same package, because when we make something `@VisibileForTesting` we rely on this.